src/components/common/scrollableAvoidKeyboard.component.tsx

```diff
@@ -19,6 +19,7 @@ class ScrollableAvoidKeyboardComponent extends React.Component<ScrollableAvoidKe
     return (
       <KeyboardAwareScrollView
         {...restProps}
+        bounces={false}
```

Comment:
You should add more restricting props and give a possibility to override them
[See an example](https://github.com/akveo/kittenTricks/blob/master/src/components/common/containerView.component.tsx#L19)

Reply:
Done